cmd/use.go

```diff
@@ -144,6 +154,16 @@ func GetDatasetRefString(f Factory, args []string, index int) (string, error) {
 	return refs[0], nil
 }
 
+// GetLinkedFilesysRef returns whether the current directory is linked to a dataset in your repo,
+// and the reference to that dataset.
+func GetLinkedFilesysRef() (bool, string) {
```

Comment:
flip the return values to follow the go "comma, ok" pattern

Reply:
Oops, correct, fixing...